packages/adblocker-benchmarks/Makefile

```diff
@@ -10,6 +10,7 @@ requests.json:
 
 ./node_modules/ubo-snfe:
 	make -C ./blockers/ublock install-nodejs clean
+	npx rollup ./node_modules/ubo-snfe/main.js --file ./node_modules/ubo-snfe/bundle.min.cjs --format cjs --context global --plugin terser
```

Comment:
@gorhill I've added the command and dependencies here so we can revert part of https://github.com/gorhill/uBlock/commit/297bcea49bf431f650dfeed515cbec6b02b00af1. I agree that it doesn't belong in the Node.js package itself, esp. because of the dependencies.